06_packages.Rmd

```diff
@@ -112,7 +112,7 @@ or online channels.
 
 
 
-## Session Info {-}
+### Session Info {-}
```

Comment:
Could we just move all session infos to the end of OMA (and exclude from individual chapters entirely)?

Reply:
Can be a separate PR, though, if easier

Reply:
should i put it in the acknowledgments chapter